### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by garak** [yohang/CalendR#66]
Well, indeed this repo has no releases.
The last tag, anyway, was published back in 2019, so it's currently not possible to install this library on PHP 8 (the only major version currently supported) unless you use dev-master (or the ^3.0@dev alias).

So, it would be nice to get a new tag **and** a new release.

**Comment by somegooser** [comment]
Yes indeed! I also would like to use this package with a new release.

**Comment by somegooser** [comment]
@yohang
